lib/challenge.js

```diff
@@ -60,6 +81,16 @@ var some_pet;
 
 var Pet;
 // Answer Starts Here
+Pet = function Pet(name, species, breed, noise){
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+};
+
+Pet.prototype.makeNoise = function makeNoise () { //I think I am misunderstanding why we need a function here if the pet noise is already a unique attribute that we defined in Q3
```

Comment:
No, you got it right!

`noise` is the property; `makeNoise` is the method that refers to it.